app/src/main/java/com/techcourse/controller/UserController.java

```diff
@@ -0,0 +1,51 @@
+package com.techcourse.controller;
+
+import com.techcourse.domain.User;
+import com.techcourse.repository.InMemoryUserRepository;
+import com.techcourse.repository.UserRepository;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import java.util.Optional;
+import nextstep.mvc.view.JsonView;
+import nextstep.mvc.view.JspView;
+import nextstep.mvc.view.ModelAndView;
+import nextstep.web.annotation.Controller;
+import nextstep.web.annotation.RequestMapping;
+import nextstep.web.support.RequestMethod;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Controller
+public class UserController {
+
+    private static final Logger log = LoggerFactory.getLogger(UserController.class);
+
+    private final UserRepository userRepository;
+
+    public UserController() {
+        this.userRepository = new InMemoryUserRepository();
+    }
+
+    public UserController(final UserRepository userRepository) {
+        this.userRepository = userRepository;
+    }
+
+    @RequestMapping(value = "/api/user", method = RequestMethod.GET)
+    public ModelAndView show(HttpServletRequest request, HttpServletResponse response) {
+        final String account = request.getParameter("account");
+        if (account == null) {
```

Comment:
```suggestion
        if (Objects.isNull(account)) {
```
위에처럼 Objects의 메서드를 사용하지 않으신 이유가 있을까요?